system_metrics_collector/src/system_metrics_collector/periodic_measurement_node.cpp

```diff
@@ -23,8 +22,9 @@
 namespace system_metrics_collector
 {
 
-/* static */ constexpr const std::chrono::milliseconds PeriodicMeasurementNode::
-DEFAULT_PUBLISH_WINDOW;
+using metrics_statistics_msgs::msg::MetricsMessage;
+
+constexpr const std::chrono::milliseconds PeriodicMeasurementNode::INVALID_PUBLISH_WINDOW;
```

Comment:
This is dangerous, could we use an optional instead?

Reply:
This changes the intended behavior (0 is the default, so not published at all), so we went from safe to dangerous.

Reply:
This sentinel value is no longer relevant to the program.